crates/router/src/connector/adyen/transformers.rs

```diff
@@ -989,14 +989,23 @@ pub struct AdyenAmountWH {
     pub currency: String,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum WebhookEventCode {
+    Authorisation,
+    Refund,
+    CancelOrRefund,
+    RefundFailed,
+}
+
```

Comment:
Derive `strum::Display` and annotate with `#[strum(serialize_all = "SCREAMING_SNAKE_CASE")]` to get a display implementation.

Reply:
Done